cirq/ops/common_gates.py

```diff
@@ -808,3 +906,73 @@ def Ry(rads: float) -> YPowGate:
 def Rz(rads: float) -> ZPowGate:
     """Returns a gate with the matrix e^{-i Z rads / 2}."""
     return ZPowGate(exponent=rads / np.pi, global_shift=-0.5)
+
+
+# The Pauli X gate.
+#   [[0, 1],
+#    [1, 0]]
+X = XPowGate()
+
+
+# The Pauli Y gate.
+#   [[0, -i],
+#    [i, 0]]
+Y = YPowGate()
+
+
+# The Pauli Z gate.
+#   [[1, 0],
+#    [0, -1]]
+Z = ZPowGate()
+
+
+# The Hadmard gate.
+#   [[s, s],
+#    [-s, s]]
+# where s = sqrt(0.5).
+H = HPowGate()
+
+
+# The Clifford S gate
+#   [[1, 0],
+#    [0, i]]
+S = Z**0.5  # Clifford S gate, [[1, 0], [0, e]] where e = exp(i pi 0.5)
+
+
+# The T gate
+#   [[1, 0]
+#    [0, exp(i pi / 4)]]
+T = Z**0.25 # Clifford T gate, [[1, 0], [0, e]] where e = exp(i pi 0.25)
+
+# The controlled phase gate.
```

Comment:
"controlled Z" is clearer, since "phase" could indicate something other than -1

Reply:
Done.